Source/NSDockTile.m

```diff
@@ -109,4 +132,74 @@ - (void) display
     }
 }
 
+- (void)draw
+{
+  NSMutableDictionary *attrs;
+  NSPoint text_location;
+  NSRect disc_rect;
+  NSSize disc_size;
+  int image_width, pad;
+  NSBezierPath *p;
+  NSPoint point;
+  CGFloat radius;
+  NSSize imageSize;
+
+  if (_showsApplicationBadge)
```

Comment:
What will get drawn if this is false but a label was set?

Reply:
I fixed this, do you like it, @fredkiefer? I always composite the image.